File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+
```

Comment:
Should have sentence about the topic

You can modify footnote and endnote contents in an existing Word document.

Reply:
I have added.